app/src/main/res/drawable/product_bus.xml

```diff
@@ -2,7 +2,8 @@
         android:width="24dp"
         android:height="24dp"
         android:viewportWidth="24.0"
-        android:viewportHeight="24.0">
+        android:viewportHeight="24.0"
+        android:tint="?attr/colorOnBackground">
```

Comment:
Was there a difference between `?attr/color*` and `?color*`? if not, might be worth being consistent.

Reply:
It's the same, we should only use one way for consistency.